crypto/ec/ecp_nistz256.c

```diff
@@ -28,6 +28,7 @@
 
 #include "internal/cryptlib.h"
 #include "crypto/bn.h"
+#include "../bn/bn_local.h"
```

Comment:
not sure I am overly enthused about this include.. Would it be nicer to come up with an internal api rather than do this.  (We do this nowhere else in the code base for bn)

Reply:
It would be, and could if done well cleanly handle the n0 argument. Nothing comes to mind immediately, but I'll think about it.

Reply:
ossl_bn_mont_ctx_set(BN_MONT_CTX *ctx, int ri, const unsigned char *rr, size_t rrlen, uint32_t nlo, uint32_t nhi)

Reply:
Add the code to crypto/bn/bn_mont.c
and to the header include/crypto/bn.h